File: app/src/main/java/it/unical/mat/embasp/specializations/dlv/DLVFilterOption.java
package it.unical.mat.embasp.specializations.dlv;

import it.unical.mat.embasp.base.OptionDescriptor;

public class DLVFilterOption extends OptionDescriptor {
    public DLVFilterOption(String initial_option) {
        super("-filter="+initial_option);
    }
}

File: CSharp version/it/unical/mat/embasp/specializations/dlv/DLVFilterOption.cs
namespace it.unical.mat.embasp.specializations.dlv
{
    using ASPFilterOption = it.unical.mat.embasp.languages.asp.ASPFilterOption;

    public class DLVFilterOption : ASPFilterOption
    {
        public DLVFilterOption(string initial_option) : base() => options += initial_option;
    }
}